source/menus/tools_menu.cpp

```diff
@@ -97,6 +98,6 @@ void ToolsMenu::displayZombieHoverInfo() {
     tww_sprintf(a_presses_str, "A: %d", GZ_getAPressesPerWindow());
     tww_sprintf(b_presses_str, "B: %d", GZ_getBPressesPerWindow());
 
-    Font::GZ_drawStr(a_presses_str, 450.f, 320.f, 0x00CC00FF, g_dropShadows);
-    Font::GZ_drawStr(b_presses_str, 450.f, 340.f, 0xCC0000FF, g_dropShadows);
+    Font::GZ_drawStr(a_presses_str, 450.f, 320.f, ColorPalette::GREEN, g_dropShadows);
+    Font::GZ_drawStr(b_presses_str, 450.f, 340.f, ColorPalette::RED, g_dropShadows);

```

Comment:
Making the zombie hover tool's color dynamically update based on mashing rate will be handled in my next pull request.